neuromation/cli/utils.py

```diff
@@ -458,6 +460,8 @@ def volume_to_verbose_str(volume: Volume) -> str:
     try:
         jobs = await client.jobs.list(name=id_or_name, owners={owner})
     except Exception as e:
+        if raise_error:
+            raise e
```

Comment:
I would remove this change: the use-case of `resolve_job()` for `port-forward` is the same as for all other commands, thus there should be no difference in its logic. Moreover, if we re-raise the exception `e` caught from `client.jobs.list()`, it will be confusingly printed directly to the console.

Reply:
Removed